src/library-authoring/add-content/AddContentContainer.tsx

```diff
@@ -80,15 +81,21 @@ const AddContentContainer = () => {
 
   const [isAddLibraryContentModalOpen, showAddLibraryContentModal, closeAddLibraryContentModal] = useToggle();
 
-  const parsePasteErrorMsg = (error: any) => {
-    let errMsg: string;
+  const parseErrorMsg = (
+    error: any,
+    detailedMessage: MessageDescriptor,
+    defaultMessage: MessageDescriptor,
+  ) => {
     try {
-      const { customAttributes: { httpErrorResponseData } } = error;
-      errMsg = JSON.parse(httpErrorResponseData).block_type;
```

Comment:
Note: this line was erroring out, because the error response didn't (ever?) contain a `block_type`. So when I pasted a component and was over the limit, the toast would disappear without showing me an error at all. This refactor ensures the error message is shown.

Reply:
> because the error response didn't (ever?)

It does return `block_type` [here](https://github.com/openedx/edx-platform/blob/bb351c4bce37b16f151431df092d4f7ebde4dd25/openedx/core/djangoapps/content_libraries/views.py#L535-L538) but it seems like the use of `IncompatibleTypesError` exception was removed now. I cannot find any place where it is raised.